### PR TITLE
Remove unnecessary indirection in Task.find/2

### PR DESCRIPTION
**Pull request by eksperimental**

### CODE REVIEW

File: lib/elixir/lib/task.ex [hunk 1]
```diff
@@ -434,11 +434,8 @@ defmodule Task do
   @doc false
   # TODO: Remove on 2.0
   # (hard-deprecated in elixir_dispatch)
-  def find(tasks, msg) do
-    do_find(tasks, msg)
-  end
-
-  defp do_find(tasks, {ref, reply}) when is_reference(ref) do
+  def find(tasks, msg)
```
Comment (kelvinst) [inline lib/elixir/lib/task.ex]: I guess you actually don't need this header definition, am I right?

Comment (eksperimental) [inline lib/elixir/lib/task.ex]: I added later thinking `h` command in IEx could show some information, but it doesn't. So I guess you are correct.

Comment (eksperimental) [inline lib/elixir/lib/task.ex]: removed! thank you @kelvinst

Comment (kelvinst) [inline lib/elixir/lib/task.ex]: ❤️ 💙 💜 💚

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**